src/client/Makefile.am

```diff
@@ -18,8 +18,6 @@ install-cockpit-client-symlink:
 	mkdir -p $(DESTDIR)/$(bindir)
 	ln -sfTv $(cockpitclientdir)/cockpit-client $(DESTDIR)/$(bindir)/cockpit-client
 
-dist_cockpitclient_SCRIPTS += src/client/cockpit-client-ssh
-
 if WITH_PYBRIDGE
```

Comment:
This isn't quite right.  We're now effectively introducing a hard dependency from the client on pybridge being enabled, which until now has been two separate options in the build system.  We need either a warning for `--enable-client` without `--enable-pybridge`, or we need to make one imply the other.

We may also consider when the correct type is to make pybridge the default, and move over to (for example) RHEL having to say `--with-old-bridge` or so.

Reply:
Good catch, thanks! I enforced this in configure.ac now. Let's also turn this around in a follow-up, I filed https://issues.redhat.com/browse/COCKPIT-1037 . I'll do this after this PR lands.